main.go

```diff
@@ -14,12 +14,12 @@ import (
 	"path"
 )
 
-type GcloudKeyFile struct {
+type gcloudKeyFile struct {
```

Comment:
Indeed :( Although the panic in the log is most likely caused by something like this, a non exported struct or similar which can't be accessed by the serializer package. Not 100% sure, but that definitely might be a problem :)